integration/tests/bft/passthrough/suite.go

```diff
@@ -0,0 +1,186 @@
+// - Two collector nodes
+// - Four consensus nodes
+// - two corrupted execution node
+// - One corrupted verification node
+// - One ghost node (as an execution node)
+func (s *Suite) SetupSuite() {
+	logger := unittest.LoggerWithLevel(zerolog.InfoLevel).With().
+		Str("testfile", "suite.go").
+		Str("testcase", s.T().Name()).
+		Logger()
+	s.log = logger
+
+	blockRateFlag := "--block-rate-delay=1ms"
+
+	s.nodeConfigs = append(s.nodeConfigs, testnet.NewNodeConfig(flow.RoleAccess, testnet.WithLogLevel(zerolog.FatalLevel)))
+
+	// generate the four consensus identities
+	s.nodeIDs = unittest.IdentifierListFixture(4)
+	for _, nodeID := range s.nodeIDs {
+		nodeConfig := testnet.NewNodeConfig(flow.RoleConsensus,
+			testnet.WithID(nodeID),
+			testnet.WithLogLevel(zerolog.FatalLevel),
+			testnet.WithAdditionalFlag("--hotstuff-timeout=12s"),
+			testnet.WithAdditionalFlag("--required-verification-seal-approvals=1"),
+			testnet.WithAdditionalFlag("--required-construction-seal-approvals=1"),
+			testnet.WithAdditionalFlag(blockRateFlag),
+		)
+		s.nodeConfigs = append(s.nodeConfigs, nodeConfig)
+	}
+
+	// generates one corrupted verification node
+	s.verID = unittest.IdentifierFixture()
+	verConfig := testnet.NewNodeConfig(flow.RoleVerification,
+		testnet.WithID(s.verID),
+		testnet.WithLogLevel(zerolog.WarnLevel),
+		testnet.AsCorrupted())
+	s.nodeConfigs = append(s.nodeConfigs, verConfig)
+
+	// generates two corrupted execution nodes
+	s.exe1ID = unittest.IdentifierFixture()
+	exe1Config := testnet.NewNodeConfig(flow.RoleExecution,
+		testnet.WithID(s.exe1ID),
+		testnet.WithLogLevel(zerolog.InfoLevel),
+		testnet.AsCorrupted())
+	s.nodeConfigs = append(s.nodeConfigs, exe1Config)
+
+	s.exe2ID = unittest.IdentifierFixture()
+	exe2Config := testnet.NewNodeConfig(flow.RoleExecution,
+		testnet.WithID(s.exe2ID),
+		testnet.WithLogLevel(zerolog.InfoLevel),
+		testnet.AsCorrupted())
+	s.nodeConfigs = append(s.nodeConfigs, exe2Config)
+
+	// generates two collection node
+	coll1Config := testnet.NewNodeConfig(flow.RoleCollection,
+		testnet.WithLogLevel(zerolog.FatalLevel),
+		testnet.WithAdditionalFlag(blockRateFlag),
+	)
+	coll2Config := testnet.NewNodeConfig(flow.RoleCollection,
+		testnet.WithLogLevel(zerolog.FatalLevel),
+		testnet.WithAdditionalFlag(blockRateFlag),
+	)
+	s.nodeConfigs = append(s.nodeConfigs, coll1Config, coll2Config)
+
+	// Ghost Node
+	// the ghost node's objective is to observe the messages exchanged on the
+	// system and decide to terminate the test.
+	// By definition, ghost node is subscribed to all channels.
+	s.ghostID = unittest.IdentifierFixture()
+	ghostConfig := testnet.NewNodeConfig(flow.RoleExecution,
+		testnet.WithID(s.ghostID),
+		testnet.AsGhost(),
+		testnet.WithLogLevel(zerolog.FatalLevel))
+	s.nodeConfigs = append(s.nodeConfigs, ghostConfig)
+
+	// generates, initializes, and starts the Flow network
+	netConfig := testnet.NewNetworkConfig(
+		"wintermute_tests",
```

Comment:
```suggestion
		"bft_passthrough_test",
```
This isn't testing Wintermute, is it?